wire-library/wire-kotlin-generator/src/main/java/com/squareup/wire/kotlin/KotlinGenerator.kt

```diff
@@ -104,6 +104,14 @@ class KotlinGenerator private constructor(
     get() = schema.getType(this) is EnumType
   private val ProtoType.isMessage
     get() = schema.getType(this) is MessageType
+  private val ProtoType.isStruct
+    get() = this == ProtoType.STRUCT_MAP
+  private val ProtoType.isListValue
+    get() = this == ProtoType.STRUCT_LIST
+  private val ProtoType.isValue
+    get() = this == ProtoType.STRUCT_VALUE
+  private val ProtoType.isNullValue
```

Comment:
should we use the same words? isStructNull, isStructValue, isStructMap, isStructList ?

isValue isn’t obvious